applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -138,118 +77,52 @@ struct encoded_audio {
 };
 
 /**
- * @brief	Generic function for a user defined button press.
- *
- * @param[in]	action	User defined action.
- *
- * @return	0 for success, error otherwise.
- */
-int le_audio_user_defined_button_press(enum le_audio_user_defined_action action);
-
-/**
- * @brief	Get configuration for audio stream.
- *
- * @param[out]	bitrate		Pointer to the bitrate used; can be NULL.
- * @param[out]	sampling_rate	Pointer to the sampling rate used; can be NULL.
- * @param[out]	pres_delay	Pointer to the presentation delay used; can be NULL.
- *
- * @retval	0		Operation successful.
- * @retval	-ENXIO		The feature is disabled.
- * @retval	-ENOTSUP	The feature is not supported,
- * @retval	error		Otherwise
- */
-int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate, uint32_t *pres_delay);
-
-/**
- * @brief	Set pointer to the connection.
- *
- * @note	Used by CIS.
- *
- * @param[in]	conn	The connection pointer.
- */
-void le_audio_conn_set(struct bt_conn *conn);
-
-/**
- * @brief	Set periodic advertising sync.
+ * @brief	Stop the LE Audio role given by @p role.
```

Comment:
What does @p do here?

Reply:
Doxygen magic, Gzregorz wanted me to add them in another PR. When you mention a parameter in the function you add @p before it.